<?xml version="1.0"?>
<launch>

  <arg name="pioneer_number" default="1" />

  <group ns="pioneer$(arg pioneer_number)">
	  <node name="each_visualizer" pkg="laserscan_visualizer" type="each_visualizer" output="screen">
      <remap from="/camera/rgb/image_raw" to="/pioneer$(arg pioneer_number)/camera/rgb/image_raw"/>
	  </node>
  </group>

</launch>

